edx_rest_framework_extensions/tests/test_permission.py

```diff
@@ -27,3 +36,65 @@ def test_has_permission_with_invalid_users(self, user):
         request.user = user
         permission = IsSuperuser()
         self.assertFalse(permission.has_permission(request, None))
+
+
+@ddt.ddt
+class JwtHasScopeTests(TestCase):
```

Comment:
Can you add a test somewhere to make sure the `dict` object in the `message` field gets properly serialized and sent in the error response?